tests/disjoint-union-perf/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -84,13 +84,14 @@ export type TypedNode =
  * @flow
  */
 
-export type InferredType = \"unknown\" |
-  \"gender\" |
-  \"enum\" |
-  \"number-or-string\" |
-  \"number\" |
-  \"string\" |
-  \"error\";
+export type InferredType = 
```

Comment:
There is a trailing space after the `=`, and it wasn't clear to me how I could get rid of it...